see-also/quick-launch-on-hybrid-net.md

```diff
@@ -0,0 +1,73 @@
+---
+description: A guide to help you run the yagna daemon with the New Network Driver enabled.
+---
+
+# Quick launch on New Network Driver
+
+### Installation
+
+We have enabled providers and requestors to run their nodes with the New Network Driver.
```

Comment:
This doesn't sound like installation instructions but rather like an introduction...